components/dropdown/test/dropdown-content.visual-diff.html

```diff
@@ -119,6 +120,14 @@
 			</d2l-dropdown-content>
 		</d2l-dropdown>
 
+		<d2l-dropdown id="min-height">
+			<div class="d2l-dropdown-opener"></div>
+			<d2l-dropdown-content min-height="1000" no-auto-fit>
```

Comment:
With just `no-auto-fit` this would have opened above, but by adding `min-height` this should open below.

Reply:
And I guess `<d2l-dropdown-content style="min-height: 1000px">` doesn't quite work? Kind of unfortunate to have attributes that mirror existing CSS things.

Reply:
Sadly it doesn't. The dropdown actually disappears when that's set which seems a bit weird.